assets/js/modules/tagmanager/datastore/settings.test.js

```diff
@@ -210,6 +210,24 @@ describe( 'modules/tagmanager settings', () => {
 					expect( console ).toHaveErrored();
 				} );
 
+				it( 'clears errors from past dispatches', async () => {
```

Comment:
I'm not sure this is needed since each of its inner actions is a fetch request which will clear its own errors before making the request again.